VMBackup/main/logbackup.py

```diff
@@ -0,0 +1,48 @@
+import os
+import re
+import sys
+import subprocess
+import threading
+from workloadPatch.logbackupPatch import logbackup
+from time import sleep
+from datetime import datetime
+
+def parameterFileParser():
```

Comment:
@shridpant - add main method and add shebang

Reply:
move into Oracle utilities with OraLogBackup as name

Reply:
Done. It is named "OracleLogBackup.py"

Reply:
Done. It is named "OracleLogBackup.py"

Reply:
Done. It is named "OracleLogBackup.py"